src/exceptions.rs

```diff
@@ -123,7 +123,7 @@ macro_rules! import_exception {
                                 ::std::panic!("Can not import module {}: {}\n{}", stringify!($module), err, traceback);
                             });
                         let cls = imp.getattr(stringify!($name)).expect(concat!(
-                            "Can not load exception class: {}.{}",
```

Comment:
TBH I would totally be happy to see "Fixed" entry in the changelog for this :)

Reply:
Ok, I can do one if you prefer.

Reply:
Done